### DISCUSSION TITLE
feat: support clickable relative filepaths

### DISCUSSION THREAD

**Discussion by bashbunni** [charmbracelet/glamour#398] [Ideas]
This is generally supported by markdown readers to include a relative filepath (e.g. `[read more](./more-info.md)`) which provides a clickable link to open the other file. 

Tested with Joplin and Neovim [Markdown Reader](https://github.com/iamcco/markdown-preview.nvim) both of which support this behaviour (though the markdown reader in neovim is limited to the selected buffer, so doesn't *quite* work)

(*[mentioned](https://discord.com/channels/1032368467246075914/1132649216402018384/1132649216402018384) in the Discord*)

**Comment by skatkov** [comment]
What would be the desired behavior in case it's not a relative filepath? It could be an URL

**Comment by bashbunni** [comment]
@skatkov yeah still clickable :)
